packages/dx-react-scheduler/docs/reference/drag-drop-provider.md

```diff
@@ -0,0 +1,77 @@
+# DragDropProvider Plugin Reference
+
+A plugin that enables users to edit appointments via drag-and-drop.
+
+## Import
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { DragDropProvider } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+If you want to use custom components, you can import the themeless plugin:
```

Comment:
```suggestion
To use custom components, import the themeless plugin:
```

Reply:
This is a standard phrase that we use in every reference article

Reply:
Ok, but you can make it shorter.